### PR TITLE
(DOCSP-11406): Update Email Addresses page

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-11406)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11406/style-guide/style/email-addresses.html)

### CODE REVIEW

File: source/style-guide/style/email-addresses.txt [hunk 1]
```diff
@@ -4,14 +4,10 @@
...
 (IANA) reserves these domains for use in examples.
 
-Additionally, format example email addresses as bold. For example,
-**yourName@example.com**.
-
-If you need to show an actual email address, use the convention in your
-authoring environment to make the email address live.
+If you need to show an actual email address, use a `standalone hyperlink
+<https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#standalone-hyperlinks>`_
+to make the email address live.
```
Comment (atsansone) [inline source/style-guide/style/email-addresses.txt:13]: I don't think it needs anything special. See the [URL page](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11406/style-guide/style/urls.html).

File: source/style-guide/style/email-addresses.txt [hunk 2]
```diff
@@ -4,14 +4,10 @@
...
 **example.org** as the domain. The Internet Assigned Numbers Authority
 (IANA) reserves these domains for use in examples.
 
-Additionally, format example email addresses as bold. For example,
-**yourName@example.com**.
-
-If you need to show an actual email address, use the convention in your
-authoring environment to make the email address live.
+If you need to show an actual email address, use a `standalone hyperlink
+<https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#standalone-hyperlinks>`_
```
Comment (atsansone) [inline source/style-guide/style/email-addresses.txt:12]:
```suggestion
If you need to show an actual email address, use a :rst:`standalone hyperlink
</restructuredtext.html#standalone-hyperlinks>`
```

File: source/style-guide/style/email-addresses.txt [hunk 3]
```diff
@@ -4,14 +4,10 @@
 Email Addresses
 ===============
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 If you need to show an example email address, use **example.com** or
```
Comment (atsansone) [inline source/style-guide/style/email-addresses.txt:7]:
```suggestion
.. default-domain:: mongodb

.. include:: /includes/styles/corrections.rst

If you need to show an example email address, use **example.com** or
```

File: source/style-guide/style/email-addresses.txt (hunk 1 above)
Comment (melissamahoney-mongodb) [reply source/style-guide/style/email-addresses.txt:13]: Yeah, I worded this sort of obscurely. Reworded to say what I mean.

File: source/style-guide/style/email-addresses.txt (hunk 2 above)
Comment (melissamahoney-mongodb) [reply source/style-guide/style/email-addresses.txt:12]: Didn't realize we had an :rst: role. Noted.

**Pull request merged**